src/js/pages/services/DeploymentsTab.js

```diff
@@ -95,17 +96,31 @@ class DeploymentsTab extends mixin(StoreMixin) {
       <dl className="deployment-services-list flush-top flush-bottom tree-list">
         <dt className="deployment-id text-uppercase">{deployment.getId()}</dt>
         {this.renderAffectedServicesList(deployment.getAffectedServices())}
+        {this.renderStaleServicesList(deployment.getStaleServiceIds())}
       </dl>
     );
   }
 
+  renderStaleServicesList(serviceIds) {
+    return serviceIds.map(function (serviceId, index) {
+      return (
+        <dd key={`stale_${index}`}>
```

Comment:
is it possible for us to have the key as `stale_${serviceId}` rather than relying on the index?